config/config.go

```diff
@@ -600,6 +614,14 @@ func (c *RemoteWriteConfig) UnmarshalYAML(unmarshal func(interface{}) error) err
 			return errors.New("empty or null relabeling rule in remote write config")
 		}
 	}
+	for header := range c.Headers {
+		if header == "authorization" {
+			return errors.New("use basic_auth or bearer_token for the authorization header")
```

Comment:
a passive message seems better.

authorization header must be changed via the basic_auth, bearer_token, or bearer_token_file parameter.